dgraphtest/cluster.go

```diff
@@ -531,13 +547,12 @@ func (gc *GrpcClient) DropPredicate(pred string) error {
 }
 
 // Mutate performs a given mutation in a txn
-func (gc *GrpcClient) Mutate(rdfs string) (*api.Response, error) {
+func (gc *GrpcClient) Mutate(mu *api.Mutation) (*api.Response, error) {
```

Comment:
Can we do both functions?

Reply:
For now, this is fine. We can add the other function when we need it.